applications/yolo_model_deployment/README.md

```diff
@@ -6,6 +6,12 @@ This project is aiming to provide basic guidance to deploy Yolo-based model to H
     <img src="./docs/meeting.gif" width="500" height="363">
 </div>
 
+## Supported Platforms
+
+The demo [container environment](Dockerfile) supports x86 or arm64 platforms with a discrete GPU.
+
+Users targeting a Jetson platform (integrated GPU) may need to install [Jetson PyTorch packages](https://pypi.jetson-ai-lab.io/)
```

Comment:
Can we provide instructions on how to install Jetson PyTorch packages from the link provided: https://pypi.jetson-ai-lab.io/?